### PR TITLE
Chat effects fixes for threads

### PR DESCRIPTION
**Pull request by gsouquet**
Fixes https://github.com/vector-im/element-web/issues/19805
Fixes https://github.com/vector-im/element-web/issues/19803

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://619c1d29e34a100075aaeac7--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/SlashCommands.tsx [hunk 1]
```diff
@@ -139,9 +139,17 @@ export class Command {
         return this.getCommand() + " " + this.args;
     }
 
-    run(roomId: string, args: string) {
+    run(roomId: string, threadId: string, args: string) {
```
Comment (turt2live) [inline src/SlashCommands.tsx:142]: explicit member access would be great 😇

**Review by turt2live — APPROVED** [review]
does this also fix chat effects on receipt, or only on send?

**Comment by gsouquet** [comment]
It fixes both!

File: src/SlashCommands.tsx (hunk 1 above)
Comment (gsouquet) [reply src/SlashCommands.tsx:142]: I'm not sure what that would look like in this context? Could you expand a bit?

**Pull request merged**